pypeit/spectrographs/keck_lris.py

```diff
@@ -1119,8 +1165,18 @@ def default_pypeit_par(cls):
         par['scienceframe']['process']['sigclip'] = 5.
         par['scienceframe']['process']['objlim'] = 5.
 
+        # Sensitivity function defaults
+        par['sensfunc']['algorithm'] = 'IR'
+        par['sensfunc']['polyorder'] = 9
+
         return par
 
+    def get_ql_master_dir(self, file):
```

Comment:
Add doc string.  

And it would be nice to think of a way to generalize this if we are going to propogate to other spectrogrpahs

Reply:
Done and I agree.